packages/dd-trace/src/proxy.js

```diff
@@ -33,6 +27,22 @@ class LazyModule {
   }
 }
 
+function lazyProxy (obj, property, getClass, ...args) {
+  const attributes = { configurable: true, enumerable: true }
```

Comment:
is it worth reusing an object and using spread operators when you could just hardcode them ?

Reply:
Probably not, originally there were more attributes so it made more sense, I can change it.

Reply:
Changed it.